sqlcollaborative/dbatools
# Fixing named, not dynamic ported instance SPN generation

Fixes # 

Changes proposed in this pull request:
 - Fixing bug where named instance with NON-dynamic ports were getting bad SPN generation (<fqdn>:<instancename>:<port>) and instead make sure generated SPNs have <fqdn>:<port>

How to test this code: 
- For a named instance, disable dynamic ports and set static port number
- Run Test-DbaSpn for server in where instance is running
- Note code should output <fqdn>:<port> instead of <fqdn>:<instancename>:<port> and DynamicPort property of object should be $false
